rosplan_action_interface/src/BaseActionInterface.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+import rospy
+import actionlib
+from abc import abstractmethod
+from RPKnowledgeBaseLink import RPKnowledgeBaseLink
+from rosplan_dispatch_msgs.msg import ActionFeedback
+
+
+# Base action interface to be exended into actionlib, service, FSM, etc.
+# Uses actionlib states to describe status of actions.
+class BaseActionInterface:
+
+    def __init__(self, action_config):
+
+        self._action_name = action_config["name"]
+        self._interface_type = action_config["interface_type"]
+        self._action_config = action_config
+
+        self._action_dispatch_msg = {}
+        self._action_status = {}
+
+    def get_action_name(self):
+        return self._action_name
+
+    def get_interface_type(self):
+        return self._interface_type
+
+    def get_plan_id(self):
```

Comment:
I don't think that this function is used anymore

Reply:
Removed.